desktop/sources/scripts/library.js

```diff
@@ -224,6 +226,18 @@ function Library (ronin) {
 
   this.cos = Math.cos
 
+  this.ln = Math.log // caclulates on the base of e
```

Comment:
you can rename this to `log`, we wont use log to print any console stuff in Ronin.